test/test-store.js

```diff
@@ -31,7 +28,11 @@ describe("hazelcast-store", function () {
       assert(typeof store === "object");
 
       HazelcastClient
-      .newHazelcastClient(clientConfig)
+      .newHazelcastClient({
+        network: {
+          clusterMembers: ['127.0.0.1:5701']
```

Comment:
note: we don't need this config actually since the default value will be enough. added since old code included it

Reply:
I'll merge and publish this tomorrow. Comment here if I  forget, I'll see the email. 

If you want to change map.remove to map.delete for performance reasons I'm ok with that as long as the tests pass. The code isn't doing anything with the returned value, so it should be fine.

Reply:
yes there is only a debug statement that logs it.

btw readme can also be changed regarding client configuration. I'll add these changes as well.